CRM/Core/Smarty/plugins/modifier.crmMoney.php

```diff
@@ -31,5 +27,11 @@
  * @throws \CRM_Core_Exception
  */
 function smarty_modifier_crmMoney($amount, $currency = NULL) {
-  return CRM_Utils_Money::format($amount, $currency);
+  if (!$amount) {
+    return $amount;
+  }
+  $currency = $currency ?? CRM_Core_Config::singleton()->defaultCurrency;
+  $money = Money::of($amount, $currency, new DefaultContext(), RoundingMode::CEILING);
+  $formatter = new \NumberFormatter(CRM_Core_I18n::singleton()->getLocale(), \NumberFormatter::CURRENCY);
```

Comment:
@eileenmcnaughton is there a reason why you didn't do something like "echo $money->formatTo('en_US');" instead?

Reply:
I read a different part of the docs? We have $currency passed in - I 'guessing' locale